File: LayoutTests/http/tests/multipart/load-last-non-html-frame.php
<?php
header('Content-type: multipart/x-mixed-replace;boundary=asdf');
?>--asdf
Content-type: text/plain

This test passes if the last multipart frame is displayed.
FAIL

<?php
# Add some padding because CFNetwork merges small multipart segments together.
echo str_pad('', 5000);
?>
--asdf
Content-type: text/plain

This test passes if the last multipart frame is displayed.
PASS
<?php
# Add some padding because CFNetwork merges small multipart segments together.
echo str_pad('', 5000);
?>--asdf--

File: LayoutTests/http/tests/multipart/load-last-non-html-frame.py
#!/usr/bin/env python3

import sys

sys.stdout.buffer.write(
    'Content-type: multipart/x-mixed-replace;boundary=asdf\r\n\r\n'
    '--asdf\n'
    'Content-type: text/plain\n'
    '\n'
    'This test passes if the last multipart frame is displayed.\n'
    'FAIL\n'
    '\n'
    '{padding}\r\n'
    '--asdf\n'
    'Content-type: text/plain\n'
    '\n'
    'This test passes if the last multipart frame is displayed.\n'
    'PASS\n'
    '{padding}\r\n'
    '--asdf--\n'.format(padding=' ' * 5000).encode()
)
